spec/features/schools/participants/add_participants/sit_add_ect_spec.rb

```diff
@@ -7,20 +7,18 @@
   include ManageTrainingSteps
 
   before do
-    outside_auto_assignment_window do
-      given_there_is_a_school_that_has_chosen_cip
-      set_participant_data
-      given_there_is_a_school_that_has_chosen_fip_and_partnered
-      and_i_am_signed_in_as_an_induction_coordinator
-      and_i_click_on(Cohort.current.description)
-      then_i_am_taken_to_fip_induction_dashboard
-
-      set_dqt_validation_result
-    end
+    given_there_is_a_school_that_has_chosen_cip
+    set_participant_data
+    given_there_is_a_school_that_has_chosen_fip_for_current_and_next_cohorts_and_partnered
+    and_i_am_signed_in_as_an_induction_coordinator
+    and_i_click_on(Cohort.current.description)
+    then_i_am_taken_to_fip_induction_dashboard
+
+    set_dqt_validation_result
   end
 
   scenario "SIT aborts process as it can't add themself as ECT" do
-    outside_auto_assignment_window do
+    inside_registration_window do
```

Comment:
@ltello I'm concerned that a handful of feature specs now need to be inside the registration window and have the next cohort set up otherwise they end up with the _registration not yet open_ step and fail. Does this seem right?

Reply:
why?

Reply:
@ltello  For the most part, the failing specs dealt with adding mentors.

If the specs are left in the timeframe of `outside_auto_assignment_window` we end up at the _registration not yet open step_. So the expectations of the tests fail as the user effectively ends on a terminating step and doesn't proceed through the wizard journey. 
This is because we are not inside the registration period for the next cohort and in most cases we are dealing with a mentor participant. This is consistent with the changes made I think.

If the specs are put in the timeframe of `inside_auto_assignment_window` most of them pass as the time sensitive clause in `BaseWizard#automatically_assign_next_cohort?` doesn't apply.

If the specs are put in the timeframe of `inside_registration_window` and the setup is done for the next cohort, then they will end up in the next cohort.

Reply:
No worries @steventux.
Cover the more cases you can and let's move forward. We need this done to continue with other stuff.
Also we'll let Matt to manually test as many scenarios as he can

Reply:
I think there's a couple of scenarios I can add which would cover arriving at the _registration not yet open_ step and the _need training setup_ step. I'm not sure it's necessary for all the specs which have changed but it would be good to test that these steps are visited and not bypassed completely. Will check.

Reply:
@ltello I've added https://github.com/DFE-Digital/early-careers-framework/pull/4855/commits/63bfb29e6b0eba0895ea6882f03e5069993544ae and https://github.com/DFE-Digital/early-careers-framework/pull/4855/commits/509fa05dd1f332933630d7375807823f1959fd25 in an attempt to cover routing to the two other steps. Not 100% sure if the spec setup for _training setup needed_ is actually representative.